### ISSUE TITLE
Settings appears once in metadata, but each set of software may have its own

### ISSUE THREAD

**Issue by Ichoran** [openworm/tracker-commons#104]
The `settings` field of Metadata should be moved to Software so different software packages can set and maintain their own settings without the user having to pry the information out of an ad-hoc JSON array or somesuch.

**Comment by MichaelCurrie** [comment]
This makes sense to me.  I'll work on that for the next release.

**Comment by Ichoran** [comment]
This change is included in the WCON specification updates in #146

**Comment by MichaelCurrie** [comment]
So the new schema for the "software" object is, to use an example for clarity, either like the below example or an array of such objects:

```
"software": 
	{
		"tracker": {"name": "MichaelTracker", "version": "0.3.4"},
		"featureID": "@OMG",
		"settings": {}
	}
```

`"settings"` can be anything desired.

I added a stub test to at least check that including settings doesn't cause a problem with an arrayed set of software specifications.  Nothing else needed to be done to the Python implementation.  So I will close this issue now.  b44a535c3c4c6274aedc0db98e61421cf7a3fbf8
